metrics/go_metrics.go

```diff
@@ -504,6 +508,9 @@ func registerAdapterMetrics(registry metrics.Registry, adapterOrAccount string,
 	am.ConnCreated = metrics.GetOrRegisterCounter(fmt.Sprintf("%[1]s.%[2]s.connections_created", adapterOrAccount, exchange), registry)
 	am.ConnReused = metrics.GetOrRegisterCounter(fmt.Sprintf("%[1]s.%[2]s.connections_reused", adapterOrAccount, exchange), registry)
 	am.ConnWaitTime = metrics.GetOrRegisterTimer(fmt.Sprintf("%[1]s.%[2]s.connection_wait_time", adapterOrAccount, exchange), registry)
+	am.ConnDialCreated = metrics.GetOrRegisterCounter(fmt.Sprintf("%[1]s.%[2]s.connection_dials", adapterOrAccount, exchange), registry)
```

Comment:
Should be something like `connection_dial_err`. Non plural to match `connection_dial_time`.